prov/efa/src/rdm/efa_rdm_msg.c

```diff
@@ -850,7 +854,6 @@ struct efa_rdm_ope *efa_rdm_msg_alloc_rxe_for_tagrtm(struct efa_rdm_ep *ep,
 		}
 		(*pkt_entry_ptr)->ope = rxe;
 
-		peer_rxe->size = efa_rdm_pke_get_rtm_msg_length(*pkt_entry_ptr);
```

Comment:
I forget why I added this earlier... is it because tag matching also need size to support `fi_peek`? If so, is it safe to remove this?

Reply:
https://github.com/ofiwg/libfabric/commit/d865260e0b2c0d6db7e487a2a4747fb0cdbd6d40